flocker/restapi/_infrastructure.py

```diff
@@ -210,6 +222,22 @@ def deco(original):
         @_logging
         @_serialize(outputValidator)
         def loadAndDispatch(self, request, **routeArguments):
+            # If-Matches support:
+            if request.requestHeaders.hasHeader(b"if-matches"):
```

Comment:
:question: I see that `hasHeader` is case-insensitive, but maybe supply `If-Matches` for consistency. I had to look it up in the docs. Others probably will too: http://twistedmatrix.com/documents/current/api/twisted.web.http_headers.Headers.html#hasHeader